modules/manage/pages/manage-xdcr/create-xdcr-replication.adoc

```diff
@@ -137,6 +141,50 @@ image::manage-xdcr/xdcr-advanced-settings.png[,400,align=left]
 The values displayed in the fields are defaults, which can be modified interactively, and saved: this may help in achieving optimal replication-performance.
 For details on the significance of each field, see the xref:xdcr-reference:xdcr-reference-intro.adoc[XDCR Reference].
 
+[#identify-target-cluster-in-xdcr]
+=== Identify the Target Cluster in a Replication
+
+A cluster is identified as a target cluster in XDCR when it receives replicated data from a remote cluster.
+Then the replicating remote cluster is identified as a source cluster.
+This is recognized as target awareness and is used to display incoming replications on the target cluster.
+
+On the XDCR UI, the target cluster displays incoming replications, showing the source cluster and bucket information for each active replication.
```

Comment:
On the XDCR UI, if the cluster is a target for XDCR replications from remote clusters, the Incoming Replications section will list the incoming replications information, grouped by the remote cluster name.  As shown in the picture below, you can open the accordion using the arrow icon to display the details of any replications incoming from the remote cluster.  There may not be any replications listed if the remote cluster has not yet created any replication, but the remote cluster name is listed because the remote cluster has added this cluster's connection info to its cluster reference list.